classes/class-alerts-list.php

```diff
@@ -338,6 +338,12 @@ public function enqueue_scripts() {
 			return;
 		}
 
+		/**
+		 * Remove the WP core tags - suggest script on the alerts list screen.
+		 * This is not needed, but more importantly it was breaking the select all functionality.
+		 */
+		wp_deregister_script( 'tags-suggest' );
```

Comment:
<!-- highlight specific problems -->
**❗️issue**
This causes the Alerts screen to not work for me. Which version of WP are you on? I've tested on 6.7.1 and 6.6.2.

Reply:
OMG, great catch 🤦🏻‍♂️. I have reverted that and added a few lines of JS to re-enable the select all toggle.